src/main/java/com/facebook/openwifirrm/modules/ProvMonitor.java

```diff
@@ -181,6 +203,40 @@ protected void syncDataToProv(
 					);
 					cfg.enableRRM =
 						cfg.enableConfig = cfg.enableWifiScan = true;
+
+					// read the details from the config
+					RRMDetails details = rrmDetails.get(serialNumber);
+					if (details == null) {
+						logger.error(
+							"No RRM details available for {} even though it has RRM enabled",
+							serialNumber
+						);
+						continue;
+					}
+
+					cfg.schedule = new RRMSchedule();
+					if (details.rrm != null) {
+						cfg.schedule.cron = details.rrm.schedule;
```

Comment:
As discussed, let's try to convert the schedule from standard cron into Quartz syntax. You can create a public static method in `RRMScheduler` for this, and maybe return null and skip/log error here if the schedule string is invalid.